src/store/actions.ts

```diff
@@ -213,7 +213,7 @@ export const clearErros = createAction(
 // FIXME: Move the rest to their own file
 export const setSettings = createAction(
   'SET_SETTINGS',
-  (settings: SettingsType) => {
+  (settings: Partial<SettingsType>) => {
```

Comment:
Could you please move this and the related reducer change (just the partial part, not the task specific part) to their own commit? It's a completely unrelated change that affects the rest of the web app.